cardano-cli/src/Cardano/CLI/Types/Transaction.hs

```diff
@@ -0,0 +1,628 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+{- HLINT ignore "Unused LANGUAGE pragma" -}
+
+module Cardano.CLI.Types.Transaction
```

Comment:
Why it's in `Types`  but it's not exporting any types?

Reply:
I was aware of this, but my priority is to get these functions out of `Cardano.CLI.Legacy.Run.Transaction`.  I've made several abortive attempts to move the implementation of the commands in `Cardano.CLI.Legacy.Run.Transaction` to `Cardano.CLI.EraBased.Run.Transaction`.  A smaller module will make this easier to do.